package.json

```diff
@@ -15,16 +15,30 @@
     "state",
     "graph"
   ],
+  "main": "./dist/homebase.react.js",
```

Comment:
I think this is also part of a separate packaging PR

Reply:
I think this is actually a bad conflict resolution from the hacky merge with master I had to do so looking into it